query_server/spi/src/query/logical_planner.rs

```diff
@@ -59,6 +59,14 @@ pub enum DDLPlan {
     CreateTable(CreateTable),
 
     CreateDatabase(CreateDatabase),
+
+    DescribeTable(DescribeTable),
+
+    DescribeDatabase(DescribeDatabase),
+
+    ShowTable(String),
+
+    ShowDatabase(),
```

Comment:
`ShowDatabases` instead of `ShowDatabase`

Reply:
nice hint